tests/subsys/zigbee/osif/nvram/src/main.c

```diff
@@ -65,7 +67,7 @@ static void test_zb_nvram_erase(void)
 	}
 }
 
-static void test_zb_nvram_write(void)
+ZTEST(osif_test, test_zb_nvram_write)
 {
```

Comment:
(un)fortunately the new ZTEST API does not allow to rely on the order of tests (they are executed alphabetically or in a random order), so this test requires a small setUp function that will erase the area before writing to it.

Reply:
Good catch, fix already applied :)